Code/Framework/AzCore/Platform/Android/platform_android_files.cmake

```diff
@@ -88,3 +88,12 @@ if (LY_TEST_PROJECT)
         VALUES LY_NO_ASSETS
     )
 endif()
+
+# Disabling -ffast-math for 'IntersectSegment.cpp' file only.
+# It has an implementation of the "Watertight Ray/Triangle Intersection"
+# algorithm, which gets broken by the fast-math optimizations.
+ly_add_source_properties(
+    SOURCES ${CMAKE_CURRENT_LIST_DIR}/../../AzCore/Math/IntersectSegment.cpp
```

Comment:
suggestion: Rather than using a relative path with (../../) could you instead use `CMAKE_SOURCE_DIR` and specify the path from the root of the repo (maybe @lumberyard-employee-dm could confirm if that's the best CMake variable to use).